rllib/agents/pg/pg.py

```diff
@@ -32,3 +86,228 @@ def get_default_config(cls) -> TrainerConfigDict:
     @override(Trainer)
     def get_default_policy_class(self, config) -> Type[Policy]:
         return PGTorchPolicy if config.get("framework") == "torch" else PGTFPolicy
+
+    @override(Trainer)
+    def training_iteration(self) -> ResultDict:
+        if self._train_trainer:
+            return self._rllib_on_train_training_iteration_fn()
+        else:
+            return super(PGTrainer, self).training_iteration()
+
+    def _rllib_on_train_training_iteration_fn(self) -> ResultDict:
+        # Some shortcuts.
+        if not self._training_workers_started:
+            for idx, w in enumerate(self._training_workers):
+                w.start.remote(0, self._num_train_workers)
```

Comment:
Should this use `idx` instead?
```suggestion
                w.start.remote(idx, self._num_train_workers)
```